ddl/column.go

```diff
@@ -1043,7 +1043,7 @@ func (w *worker) doModifyColumnTypeWithData(
 			return ver, errors.Trace(err)
 		}
 
-		reorgInfo, err := getReorgInfo(d, t, job, tbl, BuildElements(changingCol, changingIdxs))
+		reorgInfo, err := getReorgInfo(d, t, job, tbl, BuildElements(changingCol, changingIdxs), w)
```

Comment:
=> w.getReorgInfo

Reply:
added ```w.getReorgInfo```. but the old test depends on the ```getReorginfo```. it still need to refactor.

Reply:
added w.getReorgInfo. but the old test depends on the getReorginfo. it still need to refactor.